components/builder-minio/habitat/hooks/init

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+exec 2>&1
+
+{{#if cfg.use_ssl}}
+    mkdir -p {{pkg.svc_config_path}}/certs
+    ln -s {{pkg.svc_files_path}}/private.key {{pkg.svc_config_path}}/certs/private.key
```

Comment:
Should the key file names be configurable?

Reply:
They should be, but minio doesn't support that, they have to be named this specific way.